src/http/server.ts

```diff
@@ -115,6 +120,9 @@ export class Server {
         patterns,
       });
     });
+    if (options.exception) {
+      this.#exception = new options.exception();
+    }
```

Comment:
```typescript
if (options.exception) {
  this.#exception = new options.exception();
}
```
or
```typescript
if (this.#options.exception) {
  this.#exception = new this.#options.exception();
}
```
?
I hesitate ..

Reply:
this is good for now. resolving this. i say that because we check for `options.services` instead of `this.#options.services`.